### PR TITLE
Fixed apikey issue, and total seasons

### PR DESCRIPTION
**Pull request by oshribr**

### CODE REVIEW

File: omdb/client.py
Comment (dgilland) [inline omdb/client.py]: Minor nit: remove extra space before `:`.

File: omdb/models.py
Comment (dgilland) [inline omdb/models.py]: Looks like `totalResults` was also added according to http://www.omdbapi.com/ changes for 1/20/16.

Care to add that too? I don't have an apikey so I'm not able to view/verify what the search results look like now :frowning_face:.

**Review by dgilland — CHANGES_REQUESTED** [review]
Thanks for the PR! :+1: Just a few minor comments to address before merging.

File: omdb/client.py
Comment (oshribr) [inline omdb/client.py]: I'll do it. 
Should I create a new pull request or something else?

File: omdb/models.py
Comment (oshribr) [inline omdb/models.py]: It is two different think. 
The `totalResults`  is the number of the results in `search` and `pages` option. 
The `totalSeasons` is how many seasons for the request series.

File: omdb/client.py
Comment (dgilland) [inline omdb/client.py]: Same PR is fine.

File: omdb/models.py
Comment (dgilland) [inline omdb/models.py]: My suggestion was to add both `totalSeasons` and `totalResults` but because I don't have an apikey to use, I can't verify what the data returned looks like. Although, if `totalResults` is being returned at the same level as `Search`, then some other changes would be needed since omdb.py pulls the `Search` field from the response and just returns the list of results. So perhaps hold off on `totalResults` for now.

File: omdb/client.py
Comment (oshribr) [inline omdb/client.py]: I fixed only this change.

File: omdb/models.py
Comment (oshribr) [inline omdb/models.py]: My suggestion is to add `totalSeason` in this PR. 
I hope I'll find time on the weekend to fix the `totalResults`. 
I hope I'll also fix  the  #13  issue.

**Review by dgilland — APPROVED** [review]

**Pull request merged**
